src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,106 @@
+package duke;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    public ArrayList<Task> list;
+    public boolean isAlive;
+
+    public TaskList() {
+        this.list = new ArrayList<>();
+        this.isAlive = true;
```

Comment:
Try to avoid misleading names such as isAlive because as a developer reading your code, it is easy to misunderstand the meaning behind the TaskList attribute, isAlive.